ports/mimxrt10xx/supervisor/serial.c

```diff
@@ -34,12 +34,11 @@
 #include "fsl_lpuart.h"
 
 // TODO: Switch this to using DEBUG_UART.
```

Comment:
```suggestion
// TODO: Switch this to using DEBUG_UART.
```
This TODO is now done.

Reply:
I'll make one change here.